reference/unity_to_godot.rst

```diff
@@ -0,0 +1,177 @@
+| OS (export)       | | Desktop: Windows, Linux/SteamOS, OSX                                            | | Desktop: Windows, X11, OSX                                                                                   |
+|                   | | Mobile: Android, iOS, Windows Phone, Tizen,                                     | | Mobile: Android, iOS, Blackberry (deprecated)                                                                |
+|                   | | Web: WebGL                                                                      | | Web: HTML5 (via emscripten, broken)                                                                          |
+|                   | | Consoles: PS4, PS Vita, XBox One, XBox 360, WiiU, 3DS                           |                                                                                                                |
+|                   | | VR: Occulus Rift, SteamVR, Google Cardboard, Playstation VR, Gear VR, HoloLens  |                                                                                                                |
+|                   | | TV: AndroidTV, Samsung SMARTTV, tvOS                                            |                                                                                                                |
++-------------------+-----------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------+
+| Scene system      | Entity-Component System (ECS) (GameObject > Component).                           | Scene tree and nodes, allowing scenes to be nested and/or inherit other scenes                                 |
++-------------------+-----------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------+
+| Third-party tools | Visual Studio or SharpEditor                                                      | | Android SDK for Android export                                                                               |
+|                   |                                                                                   | | External editors are possible                                                                                |
++-------------------+-----------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------+
+| Killer features   | | Huge community                                                                  | | Live-editing                                                                                                 |
```

Comment:
Not sure if I would add those as killer features, as people will think Unity has this too.
I would write something like.

Scene System, Animation Pipeline, Easy to write Shaders, Integrate debugging that also runs on device.